### PR TITLE
Baseline management doc and images added

### PR DESCRIPTION
**Pull request by marstupa**
### Description

Baseline Management for Visual testing - new page and accompanying visual added

### Motivation and Context

This is to help customers understand how to navigate to and use the new feature. 

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

**Review by ebekebe — COMMENTED** [review]
Could you make sure to have at most one sentence per line, so reviewing and diffing becomes easier. For long sentences, breaking more often is also useful.

File: static/img/sauce-visual/build-overlay-history.jpg
[the remark this answers is not in the document]
Comment (marstupa) [reply static/img/sauce-visual/build-overlay-history.jpg:1]: Good question, I don't know. I replaced with another example.

File: docs/visual-testing/workflows/baseline-management.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# Baseline Management
+
+The Baseline Management feature lets you oversee the approved visual snapshots from your tests.
+It allows you to track visual regressions through time and check baselines for different branches and projects.
+Additionally, you can view the baseline history, download images, and view associated build details.
+
+While logged in to Sauce Labs, navigate to this feature by clicking the "Visual -> Baselines" in the side navigation.
```
Comment (konraddysput) [inline docs/visual-testing/workflows/baseline-management.md:13]:
```suggestion
While logged in to Sauce Labs, you can access this feature by clicking "Visual -> Baselines" in the side navigation.
```

File: docs/visual-testing/workflows/baseline-management.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+<img src={useBaseUrl('img/sauce-visual/baselines-page.jpg')} alt="Baselines page results" />
+
+## Baseline History View
+
+Baseline History allows you to view and manage historical snapshots of a single test baseline.
+This view opens after you click on any snapshot from the Baselines page.
+
+<img src={useBaseUrl('img/sauce-visual/baseline-history.jpg')} alt="Baseline history view displaying a snapshot with list of changes in time" />
+
+Once inside Baseline History view, you can perform the following actions by clicking the `...` button on any item in the list:
```
Comment (konraddysput) [inline docs/visual-testing/workflows/baseline-management.md:34]:
```suggestion
Once inside the Baseline History view, you can perform the following actions by clicking the `...` button on any item in the list:
```

**Review by konraddysput — APPROVED** [review]

**Pull request merged**